iroh-bytes/src/store/flat.rs

```diff
@@ -1173,17 +1173,27 @@ impl Store {
         Ok(())
     }
 
+    /// Path to the directory where complete files and outboard files are stored.
+    pub(crate) fn complete_path(root: &Path) -> PathBuf {
```

Comment:
These are pub(crate) intentionally since this is an internal detail. This means that some tests have to manually duplicate the code, but I think that is fine.